js/output/pjs/pjs-code-injector.js

```diff
@@ -1076,13 +1091,16 @@ class PJSCodeInjector {
                         resourceCache[filename] = img;
                     });
                 };
-
+                var findWhere = function(array, criteria) {
+                    return array.find(item => Object.keys(criteria).every(
+                        key => item[key] === criteria[key]));
+                };
                 var loadSound = function(filename) {
                     return new Promise((resolve) => {
                         var audio = document.createElement("audio");
                         var parts = filename.split("/");
 
-                        var group = _.findWhere(OutputSounds[0].groups, { groupName: parts[0] });
```

Comment:
Yes, I think so, given we only use parts of it in webapp, and that many things are now available in ES6. I did originally make a bunch of lodash/underscore removal related changes in my big branch, but then realized that wasn't the P1 change we needed and reverted them. I kept the change here since this exported code should have minimal dependencies.